1-exercises/A-accessing-values/exercise1.js

```diff
@@ -2,27 +2,28 @@
   This object has 4 properties
   The properties of the object are all primitive types (string, number or boolean)
   What is the type of each property?
+  string, string, boolean, number respectively 
 */
 
 let dog = {
   breed: "Dalmatian",
   name: "Spot",
   isHungry: true,
-  happiness: 6
+  happiness: 6,
```

Comment:
I see you added a comma here, having a comma or not is a stylistic choice but as a challenge if you have time, think of one advantage to having it with and one advantage to having it without the comma